src/Raven.Server/SqlMigration/SqlQueries.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections.Generic;
+
+namespace Raven.Server.SqlMigration
+{
+    public static class SqlQueries
+    {
+        public const string SelectAllTables = "select TABLE_SCHEMA, TABLE_NAME from INFORMATION_SCHEMA.TABLES where TABLE_TYPE = @tableType order by 'TABLE_SCHEMA','TABLE_NAME'";
```

Comment:
Avoid using `@` as a constant parameter name, have the factor generate the parameter name with the appropriate prefix for the database.